.github/workflows/ci.yml

```diff
@@ -191,6 +191,13 @@ jobs:
           gpg_private_key: ${{ secrets.CALYPTIA_GPG_KEY }}
           passphrase: ${{ secrets.CALYPTIA_GPG_KEY_PASSPHRASE }}
 
+      - name: Get the manifest
+        run: |
+          version=$(curl -s https://api.github.com/repos/calyptia/core-operator-releases/releases | jq .[0].tag_name | sed -E 's/.*"([^"]+)".*/\1/')
+          curl -sSfL https://github.com/calyptia/core-operator-releases/releases/download/${version}/manifest.yaml -o ./cmd/operator/manifest.yaml
```

Comment:
@patrick-stephens is this better?

Reply:
no idea, does it work? :)

This feels bad though to only add in CI, should there not be a local file for it we can just PR?

Reply:
Local file is required for local builds. So yes there is one. 
Maybe it is better to do this like we do when we release core-operator when we create PR in charts repo.

Reply:
yeah I think so, then we can review it works in the PR too